hook.js

```diff
@@ -217,6 +217,14 @@ function addIitm (url) {
 function createHook (meta) {
   let cachedResolve
   const iitmURL = new URL('lib/register.js', meta.url).toString()
+  let includeLibs, excludeLibs
+
+  async function initialize (options) {
```

Comment:
nit: I would actually call this `data` instead of `options` to match with the naming scheme on the module hook.